src/styles/themes/_default-theme.scss

```diff
@@ -6,201 +6,291 @@
-  --primary-color-20: #bce4ff; // $color-background2-primary (Surface 2)
-  --primary-color-30: #8ed0fa; // $color-background3-primary (Surface 3)
+  --primary-color-10: #ebf7ff;
+  --primary-color-20: #bce4ff;
+  --primary-color-30: #8ed0fa;
   --primary-color-40: #68baef;
   --primary-color-50: #47a4df;
-  --primary-color-60: #2c8cc9; // $color-tertiary (Graphics)
-  --primary-color-70: #146da6; // $color-secondary (Text)
-  --primary-color-80: #054d7b; // $color-primary (Extra high contrast)
+  --primary-color-60: #2c8cc9;
+  --primary-color-70: #146da6;
+  --primary-color-80: #054d7b;
   --primary-color-90: #002a47;
   --primary-color-100: #00060a;
-  --primary-color: #054d7b; // $color-primary (Extra high contrast)
+
+  --primary-background1-color: var(--primary-color-10); // Surface 1
+  --primary-background2-color: var(--primary-color-20); // Surface 2
+  --primary-background3-color: var(--primary-color-30); // Surface 3
+  --primary-tertiary-color: var(--primary-color-60); // Graphics
+  --primary-secondary-color: var(--primary-color-70); // Text
+  --primary-color: var(--primary-color-80); // Extra high contrast
 
   // Accent base colors
-  --accent-color-10: #ebfdff; // $color-background1-accent (Surface 1)
-  --accent-color-20: #b0f3fe; // $color-background2-accent (Surface 2)
-  --accent-color-30: #7be4f4; // $color-background3-accent (Surface 3)
+  --accent-color-10: #ebfdff;
+  --accent-color-20: #b0f3fe;
+  --accent-color-30: #7be4f4;
   --accent-color-40: #50cee1;
   --accent-color-50: #2db3c7;
-  --accent-color-60: #1999ac; // $color-tertiary-accent (Graphics)
-  --accent-color-70: #0b7b8b; // $color-secondary-accent (Text)
-  --accent-color-80: #025966; // $color-primary-accent (Extra high contrast)
+  --accent-color-60: #1999ac;
+  --accent-color-70: #0b7b8b;
+  --accent-color-80: #025966;
   --accent-color-90: #00333b;
   --accent-color-100: #00090a;
-  --accent-color: #025966; // $color-primary-accent (Extra high contrast)
+
+  --accent-background1-color: var(--accent-color-10); // Surface 1
+  --accent-background2-color: var(--accent-color-20); // Surface 2
+  --accent-background3-color: var(--accent-color-30); // Surface 3
+  --accent-tertiary-color: var(--accent-color-60); // Graphics
+  --accent-secondary-color: var(--accent-color-70); // Text
+  --accent-color: var(--accent-color-80); // Extra high contrast
 
   // Red base colors
-  --red-color-10: #ffefef; // $color-background1-red (Surface 1)
-  --red-color-20: #ffc6c6; // $color-background2-red (Surface 2)
-  --red-color-30: #ffa3a3; // $color-background3-red (Surface 3)
+  --red-color-10: #ffefef;
+  --red-color-20: #ffc6c6;
+  --red-color-30: #ffa3a3;
   --red-color-40: #f48686;
   --red-color-50: #e46f6f;
-  --red-color-60: #c45353; // $color-tertiary-red (Graphics)
-  --red-color-70: #993838; // $color-secondary-red (Text)
-  --red-color-80: #6c2222; // $color-primary-red (Extra high contrast)
+  --red-color-60: #c15151;
```

Comment:
intentional value change?

Reply:
Yes. Was not 100% in line with the Figma spec.